### PR TITLE
dockerize build

### PR DESCRIPTION
**Pull request by ztbrown**
compile lcm without needing lcm on your system, target linux, or whatever else you can think of. 

Also installs libcheck and valgrind so you can test your code and stop living like an animal.

### CODE REVIEW

**Comment by jdenen** [comment]
So will this close #1? Or this just sets up the work to close #1?

**Comment by ztbrown** [comment]
No, this would give the author and any contributors the framework for #1. Future PRs would have to address test coverage. I'd love to know what @d-- is thinking about in terms of Road Map and #2, in particular.

File: README.md [hunk 1]
```diff
@@ -10,4 +10,9 @@ pushd src
 lcm-gen -c say.lcm
 popd
 ```
+Alternatively, with Docker
 
+```bash
+docker build -t dogesay .
```
Comment (phasebash) [inline README.md]: encapsulate in a build.sh?

Comment (ztbrown) [reply README.md]: This is a good idea.

**Comment by ztbrown** [comment]
@phasebash @d-- check it 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

**Review by jdenen — APPROVED** [review]

**Pull request merged**